### ISSUE TITLE
流式 web_demo_stream.py 错误：Floating point exception (core dumped)

### ISSUE THREAD

**Issue by SevenMpp** [VITA-MLLM/VITA-Audio#15]
流式与非流式demo输入文本或语音服务崩溃，Floating point exception (core dumped)

版本：
torch: 2.3.1+cu121
cuda:  12.1
flash_attn:
flash_attn-2.7.3+cu12torch2.3cxx11abiFALSE-cp311-cp311-linux_x86_64.whl

错误信息：
非流式：
python web_demo.py 
INFO:vita_audio.tokenizer_sensevoice_glm4voice:self.rank=0
Loading checkpoint shards: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:03<00:00,  1.09it/s]
* Running on local URL:  http://0.0.0.0:7027
* To create a public link, set `share=True` in `launch()`.
[('你都可以做什么', None)]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Floating point exception (core dumped)
流式：
![Image](https://github.com/user-attachments/assets/9a1eeb83-4bb9-4611-b54a-a6ec9277421b)

**Comment by longzw1997** [comment]
看着像环境的报错，可以考虑使用我们提供的镜像
docker pull shenyunhang/pytorch:24.11-py3_2024-1224

**Comment by SevenMpp** [comment]
非常感谢您的回复，已解决

**Comment by SevenMpp** [comment]
实在 抱歉。我之前是增加了attention_mask ,解决了，出现了电子音比较重。但是目前看到的错误是上面的图片，这个是在启动的时候在预热哪个测试样例的时候就报错了。发现的原因是 这个。好像不是环境问题吧，而且主要是怀疑是因为手动增加的attention_mask 导致的电子音重。是不是？？？？


问题 | 原因
pad_token == eos_token | 导致 attention_mask 无法推断 
浮点异常 | 注意力掩码无效、计算 NaN

**Comment by SevenMpp** [comment]
> 看着像环境的报错，可以考虑使用我们提供的镜像 docker pull shenyunhang/pytorch:24.11-py3_2024-1224

我又尝试了 torch_dtype = torch.bfloat16
#torch_dtype = torch.float32
audio_tokenizer = get_audio_tokenizer(
    audio_tokenizer_path,
    audio_tokenizer_type,
    flow_path=flow_path,
    rank=audio_tokenizer_rank,
)
print("audio_tokenizer torch_dtype: ", torch_dtype)
audio_tokenizer.load_model()
将 torch.bfloat16 改成 torch.float32 就没出现上面的问题，因此是不是在前向传播的时候 计算出现了问题
